Workarounds/Splatoon_PortNumber/rules.txt

```diff
@@ -0,0 +1,59 @@
+[Definition]
+titleIds = 0005000010176900,0005000010176A00,0005000010162B00
+name = Multiplayer port
+path = "Splatoon/Workarounds/Multiplayer port"
+description = Forces the multiplayer port to be 58008.
```

Comment:
I'm sure no one will be caught off guard by this, but probably best if this matches the other packs.